ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/XARecoveryModuleUnitTest.java

```diff
@@ -441,8 +446,7 @@ public String getProductVersion() {
                 public String getJndiName() {
                     return "test";
                 }
-
-                int count = 0;
+                int recoverCount = 0;
```

Comment:
Now that you have gone back to the original use of this variable I think it should keep the original name - in the future it will make code diffs simpler to analyse.

Reply:
Thanks, updated